src/main/java/net/minecraftforge/fml/common/network/simpleimpl/SimpleNetworkWrapper.java

```diff
@@ -160,7 +160,7 @@ public SimpleNetworkWrapper(String channelName)
 
     /**
      * Construct a minecraft packet from the supplied message. Can be used where minecraft packets are required, such as
-     * {@link TileEntity#func_145844_m}.
+     * {@link TileEntity#getDescriptionPacket}.
```

Comment:
No, our comments should be in srg name as they get remapped in the end modder's setup to whatever mapping they are using.